cmd/cri-containerd/options/options.go

```diff
@@ -59,8 +59,8 @@ func (c *CRIContainerdOptions) AddFlags(fs *pflag.FlagSet) {
 		"/var/lib/cri-containerd", "Root directory path for cri-containerd managed files (metadata checkpoint etc).")
 	fs.StringVar(&c.ContainerdEndpoint, "containerd-endpoint",
 		"/run/containerd/containerd.sock", "Path to the containerd endpoint.")
-	fs.DurationVar(&c.ContainerdConnectionTimeout, "containerd-connection-timeout",
```

Comment:
Hard coded to 60seconds at the moment.  https://github.com/containerd/containerd/blob/master/client.go#L67

We should probably think about a PR to set the timeout value seems like an arbitrarily long time to wait for a connection.

Reply:
The problem is that containerd doesn't provide a way to only configure the timeout. It means that we have to customize all `dialoptions` if we want to overwrite. Given so, I think we may just use default. :)
See https://github.com/containerd/containerd/blob/master/client_opts.go#L29.

Reply:
yeah by PR I meant pr against containerd